Ginger/GingerCore/NoSqlBase/GingerCassandra.cs

```diff
@@ -413,7 +415,11 @@ public bool IfUseJSON()
 
         public override void PerformDBAction()
         {
-            Connect();
+            //if (!Connect())
+            //{
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this commented out code.](https://app.codacy.com/manual/Ginger-Automation/Ginger/pullRequest?prid=5368638)